tests/test_distributions.py

```diff
@@ -348,10 +363,23 @@ def test_rvs_test_point_are_valid(tf_seed, distribution_conditions):
     dist_class = getattr(pm, distribution_name)
     dist = dist_class(name=distribution_name, **conditions)
     test_value = dist.test_value
+    if distribution_name in ["Flat", "HalfFlat"]:
+        # pytest.skip("Flat and HalfFlat distributions don't support sampling.")
+        assert test_value.shape == dist.batch_shape + dist.event_shape
+        return
     test_sample = dist.sample()
     logp = dist.log_prob(test_value).numpy()
     assert test_value.shape == test_sample.shape
     assert tuple(test_value.shape.as_list()) == tuple(
         (dist.batch_shape + dist.event_shape).as_list()
     )
     assert not (np.any(np.isinf(logp)) or np.any(np.isnan(logp)))
+
+
+@pytest.mark.parametrize("distribution_name", ["Flat", "HalfFlat"])
+@pytest.mark.parametrize("sample", [tf.zeros(1), tf.zeros((1, 3, 4)), tf.zeros((1, 5, 3, 4))])
```

Comment:
We are using fixtures instead of mark parameterize. Could you change it to that?

Reply:
Sure!